### ISSUE TITLE
Docker image

### ISSUE THREAD

**Issue by matthias4217** [matthias4217/jukebox-ultra-nrv#76]
Could be nice and more easy to use

**Comment by Ardhanaariishvara** [comment]
Yeah that would be very nice indeed, but that's a huge workload for little results.

**Comment by matthias4217** [comment]
Support has been added with 0be0f81cf14291da9272179031be90ae6e971644. However it's probably still imperfect, and I'll be checking it to see if it really works.
